extension/manifest.json

```diff
@@ -1,8 +1,8 @@
 {
   "manifest_version": 2,
+  "name": "Apiary Chrome Extension",
   "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEArmcFCbSNikigxEi5/RX1AQ0mEQzMsqlQfikEJdkjjVbDol7UqwKIVioHvn+lWI+o5dyj6qSTXlwB7N9aNXHJFcwI61eU+aWVLB0RObmS5no8URc12GBXn30SzYa89IRSj5hWPnzaVZLlQ0JincaPfBd3fCHVFgAPMOazqv8a/RxnkpwORvhMe5hRDrMznRDCBXxGqqw7qHyflylS8E/qxBl7DkdtMV/YRUpjMUFpvt6JEOE7dLsH+TIUFrp/XCBZo+TPxNkPVxYLaU8bcbwKaQQ2PDL1dwrQSgp0meM4I2ob3jW0v2nkAyw2kwRNXWGTAr7qqzdxSYBycXdWkUP/BwIDAQAB",
-  "name": "Apiary Proxy Chrome Extension",
-  "version": "1.0.0",
+  "version": "0.0.5",
```

Comment:
If you want to start using semrel (ask Javor how awesome it is) I recommend you practicing the commit message style :+1: